🐞[Bug] - Build with Gradle 중 에러

# 🐞 Bug - Build with Gradle 중 에러

### 🕵️‍♀️ Condition

---
<!-- 아래에 버그가 발생한 상황을 적어주세요 -->
Github Action 실행 중 Build with Gradle 에서 에러가 발생한다.

### 📝 Description

---
<!-- 아래에 버그가 발생한 상황을 적어주세요 -->
```
* What went wrong:
Execution failed for task ':spotlessJavaCheck'.
> The following files had format violations:
      src/test/java/ceos/backend/domain/application/ApplicationControllerTest.java
          @@ -1,5 +1,6 @@
           package·ceos.backend.domain.application;
           
          +
           import·org.junit.jupiter.api.DisplayName;
           import·org.junit.jupiter.api.Test;
           import·org.springframework.beans.factory.annotation.Autowired;
          @@ -13,8 +14,7 @@
           @AutoConfigureMockMvc
           public·class·ApplicationControllerTest·{
           
          -····@Autowired
          -····private·MockMvc·mockMvc;
          +····@Autowired·private·MockMvc·mockMvc;
           
           ····@DisplayName("지원서·엑셀·파일·생성·시각·API·-·권한·X")
8 actionable tasks: 8 executed
           ····@Test
      src/test/java/ceos/backend/domain/application/ApplicationExcelServiceTest.java
          @@ -1,5 +1,6 @@
           package·ceos.backend.domain.application;
           
          +
           import·ceos.backend.domain.application.dto.response.GetCreationTime;
           import·ceos.backend.domain.application.service.ApplicationExcelService;
           import·org.assertj.core.api.Assertions;
          @@ -11,8 +12,7 @@
           @SpringBootTest
           public·class·ApplicationExcelServiceTest·{
           
          -····@Autowired
          -····ApplicationExcelService·applicationExcelService;
          +····@Autowired·ApplicationExcelService·applicationExcelService;
           
           ····@DisplayName("지원서·엑셀·파일·생성·시각")
           ····@Test
  Run './gradlew :spotlessApply' to fix these violations.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org/

BUILD FAILED in 27s
Error: Process completed with exit code 1.
```

### ✅ To Do List 

---
<!-- 아래에 수정 사항을 적어주세요 PR 날릴 때 모두 체크되어야함 -->
- [ ] deploy.yml 수정

--- a/.github/workflows/deploy.yml
+++ b/.github/workflows/deploy.yml
@@ -43,7 +43,7 @@ jobs:
 
       # Gradle build
       - name: Build with Gradle
-        run: ./gradlew build -x test
+        run: ./gradlew build -x test :spotlessApply
 
       - name: Docker meta
         id: docker_meta

--- a/src/main/java/ceos/backend/domain/application/domain/Pass.java
+++ b/src/main/java/ceos/backend/domain/application/domain/Pass.java
@@ -12,7 +12,7 @@ import lombok.RequiredArgsConstructor;
 public enum Pass {
     PASS("합격"),
 
-    FAIL("탈락");
+    FAIL("불합격");
 
     @JsonValue private final String result;
 

--- a/src/test/java/ceos/backend/domain/application/ApplicationControllerTest.java
+++ b/src/test/java/ceos/backend/domain/application/ApplicationControllerTest.java
@@ -1,5 +1,6 @@
 package ceos.backend.domain.application;
 
+
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
@@ -13,8 +14,7 @@ import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
 @AutoConfigureMockMvc
 public class ApplicationControllerTest {
 
-    @Autowired
-    private MockMvc mockMvc;
+    @Autowired private MockMvc mockMvc;
 
     @DisplayName("지원서 엑셀 파일 생성 시각 API - 권한 X")
     @Test

--- a/src/test/java/ceos/backend/domain/application/ApplicationExcelServiceTest.java
+++ b/src/test/java/ceos/backend/domain/application/ApplicationExcelServiceTest.java
@@ -1,5 +1,6 @@
 package ceos.backend.domain.application;
 
+
 import ceos.backend.domain.application.dto.response.GetCreationTime;
 import ceos.backend.domain.application.service.ApplicationExcelService;
 import org.assertj.core.api.Assertions;
@@ -11,8 +12,7 @@ import org.springframework.boot.test.context.SpringBootTest;
 @SpringBootTest
 public class ApplicationExcelServiceTest {
 
-    @Autowired
-    ApplicationExcelService applicationExcelService;
+    @Autowired ApplicationExcelService applicationExcelService;
 
     @DisplayName("지원서 엑셀 파일 생성 시각")
     @Test

